app.py

```diff
@@ -46,11 +48,26 @@ def jsonify(self, requestIpAddress):
             "id": self.id}
     def csvify(self):
         return f"{self.name};{self.ipAddress};{self.startTime.strftime(dateTimeFormat)};{self.endTime.strftime(dateTimeFormat)};{self.id}\n"
+    def getMinutesReserved(self):
+        beginTime = self.startTime if datetime.now() < self.startTime else datetime.now()
+        timeDelta = self.endTime - beginTime
+        return timeDelta.days * 24 * 60 * 60 + timeDelta.seconds // 60
 
 class ReservationForm(FlaskForm):
-    name = StringField("Name", validators=[DataRequired(), Length(2, 40), Regexp("^[a-zA-Z0-9 ]*$")])
-    minutes = IntegerField("Number of minutes", validators=[DataRequired(), NumberRange(min=1, max=120)])
+    name = StringField("Name", 
+            validators=[DataRequired(), Length(2, 40), Regexp("^[a-zA-Z0-9 ]*$")],
+            description="Only alphanumeric characters allowed")
+    minutes = IntegerField("Number of minutes", 
+            validators=[DataRequired(), NumberRange(min=1, max=MAX_RESERVATION_MINUTES)], 
+            description=f"All reservations combined can be a maximum of {MAX_RESERVATION_MINUTES} minutes")
```

Comment:
Add to the README that the "MAX_RESERVATION_MINUTES" cannot be 1

Reply:
Revamped to MAX_RESERVATION_TIME